### DISCUSSION TITLE
[F] hope support webp

### DISCUSSION THREAD

**Discussion by chaoshenglu** [exceljs/exceljs#2452] [Ideas & Proposals]
## 🚀 Feature Proposal

hope support webp

## Motivation

I have a large number of images to process and I need to use webp to save traffic

## Example

const imageId1 = workbook.addImage({
  filename: 'path/to/image.webp',
  extension: 'webp',
});

**Comment by Cat7373** [comment]
You can try magic, for example: `extension: 'webp' as 'jpeg'`

<img width="298" alt="image" src="https://github.com/exceljs/exceljs/assets/9296576/5d95a7c9-9a79-484e-b1b5-19382271c450">

**Comment by Cat7373** [comment]
It is more than just a psychological comfort. In fact, the generated files will also change. Although Excel seems to be able to handle incorrect image extensions correctly, other software may not be able to handle them correctly.

![image](https://github.com/exceljs/exceljs/assets/9296576/c259719d-6850-40f0-ab9e-99cd29ff53e8)
